multi-step-flow/src/app/extensions/components/MenuPanelContent.tsx

```diff
@@ -0,0 +1,167 @@
+import React, { useState } from 'react';
+import {
+  Button,
+  Flex,
+  Form,
+  Heading,
+  PanelBody,
+  PanelFooter,
+  PanelSection,
+  Table,
+  TableBody,
+  ToggleGroup,
+} from '@hubspot/ui-extensions';
+import { MenuItem, type MenuPanelContentProps } from '../types';
+import { MenuItemRow } from './MenuItemRow';
+
+export const MenuPanelContent = ({
+  restaurant,
+  onAddToCartClick,
+  closePanel,
+}: MenuPanelContentProps) => {
+  const [menuItem, setMenuItem] = useState<MenuItem>();
```

Comment:
```suggestion
  const [menuItem, setMenuItem] = useState<MenuItem | null>(null);
```
prefer null over undefined here